### ISSUE TITLE
New Goals

### ISSUE THREAD

**Issue by DevilIris** [Wolferos/Hearts-of-Iron-IV-The-Great-War#141]
NF for korea. //developing
NF for Unitde Baltic Duchy and lithunia
NF for Qing
NF for Spain
NF for Greece 
NF for Brazil//Developing
NF for Sweden 
NF for Finland//developing
Expand NF for Japan to the vanilla focus tree 
Expand Soviet Union Focus tree with the vanilla one
Rework Peace events
Rework AI strategies
Make new 3d Models for major nations
Add more Loading Screens as well non-copyright music
Add more generals including add generic generals. (Also a event that remove the staff to the new generals of ww2)

Add more states and improve the map.(Citofis is doing that)

Add more Laws? Maybe what do you think?
![20776529_207547893111740_9097475441552767359_o](https://user-images.githubusercontent.com/24885732/29554892-72450114-86e6-11e7-8a03-97b27abaacdb.jpg)

**Comment by MasterWaifuman** [comment]
> Add more Laws? Maybe what do you think?

I dunno. You have ideas?

**Comment by DevilIris** [comment]
Tax system? Or trade system?

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MasterWaifuman** [comment]
it can be for the next big update.

**Comment by DevilIris** [comment]
Also the new goal it could be finish the colonial nations of the other issue post of kaiser annony

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MasterWaifuman** [comment]
maybe i could work on colonies

**Comment by DevilIris** [comment]
Kaiser research all of that but we need to code that and is basically make a new country per nation. 

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MasterWaifuman** [comment]
yeah i can do it

**Comment by MasterWaifuman** [comment]
also for next major update

**Comment by DevilIris** [comment]
I am going to talk with @Citofis so he can make changes to the map.

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by MasterWaifuman** [comment]
also people on steam are insane about crashes so can you tell them that i fixed it and it will come with new update soon ( i can't comment on steam ( dunno why ) )

**Comment by MasterWaifuman** [comment]
@DevilIris, that thing you did to limit AI training divisions, can you limit concretely Belgium to like 25 divisions or so, because 150 divisions for Belgium is way too much.

**Comment by DevilIris** [comment]
@MasterWaifuman  Is nor possible fo only one nation needs to be for all the nations and is a define

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by skullbe** [comment]
not a only NF. Historical ideas, country leaders too. And current ideology have serious problem. we need one more ideology 'monarchy'. because i dont know what is different between authoritarian and fascism. For example,  In Vanilla japan is fascism but TGW is authoritarian. government system and leader(yoshihito) are same. i dont think need more work to make this. just modding, and replace ideology. non alliance should be non alliance.

**Comment by skullbe** [comment]
*monarchy

**Comment by skullbe** [comment]
i will try to make this and send files to devilliris for testing.

**Comment by DevilIris** [comment]
Also too many people want to boost the non allign via a advisor reformer we could add that.

_Sent from my Samsung SM-G532M using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by skullbe** [comment]
http://imgur.com/a/GtJMQ

how about this?

**Comment by MasterWaifuman** [comment]
> Rework Peace events

Versailles and Frankfurt reworked ( #146 )

**Comment by MasterWaifuman** [comment]
> Rework AI strategies

Fixed AI for GER ( #146 )
Working on BEL

**Comment by DevilIris** [comment]
Sorry for being AFK i was busy with the school and my work i cant effort to help in this right now the next weekend i will have another major update in fixes and other stuff. BTW thx @MasterWaifuman  to keep working on this. Wolferos didn't merge this yet because h only update the next month

**Comment by MasterWaifuman** [comment]
> we need one more ideology 'monarchy'.

I don't think that we need a brand new ideology. We can just add sub-ideology for authoritarian, like nazism sub-ideology for fascism. Because if we add new ideology in the middle of development it will cause a lot of problems. Adding new ideologies and everything that can be globally affected should have been done as one of the first things in development.

**Comment by MasterWaifuman** [comment]
> i dont know what is different between authoritarian and fascism

As the fascism is pretty much totalitarianism, there are the differences
![image](https://user-images.githubusercontent.com/30695036/31096449-b9bf0d24-a7c4-11e7-9b99-7a98d55dcf54.png)

**Comment by MasterWaifuman** [comment]
Also, I will end with SOV focus tree for now. I will make fixes for it if people find something. Now my main direction is fixing BEL and FRA AI (because as shown my research GER cannot push through BEL because of French troops in BEL). I have a couple of thoughts for GER focus tree and maybe Spain, but these are secondary.

**Comment by DevilIris** [comment]
What about the colonial nation with the dominions and puppets

**Comment by MasterWaifuman** [comment]
I'm working on it. I already made Togoland. Now going to make Cameroon.

**Comment by MasterWaifuman** [comment]
I have a lot of work in school, so I will make one colony per weekend. But other things like small fixes I hope to do as usual. Also, I fixed BEL unit spam. I tested it in 1914 scenario. FRA always moves her troops into Belgium, so it is pretty hard to break through. But in this scenario Germany default army, so it would be easier if you start in 1910. Also, I removed bunkers from french borders and increased number of trenches

**Comment by MasterWaifuman** [comment]
I figured that BEL AI fix doesn't work. In 1914 scenario BEL has no equipment so it can't spam troops. I will balance all this through national spirits and events

**Comment by MasterWaifuman** [comment]
To @Wolferos:
I think a mod should be updated in a few days. I fixed a lot of stuff and soon will make the last commit for the current update. All in commit #146.

**Comment by MasterWaifuman** [comment]
Is this mod still alive? There were no updates for a long time.

**Comment by DevilIris** [comment]
Yea  it is alive but the most of the dev. Is busy with school etc. I want
to talk to wolferos to make another people the dev in steam do we can
update more the mod

**Comment by MasterWaifuman** [comment]
I want to add a new thing to GER: event that allows restructuring AUH. I can make it instead of "Bavarian sth" because I think this focus quite useless. I, for example, never take this focus. It gives too little but takes too much. What do you think?

**Comment by DevilIris** [comment]
wait. dont do more work cuz gerrade is making a new states

**Comment by DevilIris** [comment]
he is rebuilding all the map
